tornado/platform/asyncio.py

```diff
@@ -576,7 +576,21 @@ def _run_select(self) -> None:
                         raise
                 else:
                     raise
-            self._real_loop.call_soon_threadsafe(self._handle_select, rs, ws)
+
+            try:
+                self._real_loop.call_soon_threadsafe(self._handle_select, rs, ws)
+            except RuntimeError:
+                # "Event loop is closed". Swallow the exception for
+                # consistency with PollIOLoop (and logical consistency
+                # with the fact that we can't guarantee that an
+                # add_callback that completes without error will
+                # eventually execute).
+                pass
```

Comment:
Should the exception message be inspected (or something else be checked) here to ensure that it wasn't some other `RuntimeError`?

Reply:
Maybe, but there are no guarantees that the error message won't be changed. I wish there was a more specific error class we could catch, but I'm not too worried about being overly-broad here in practice.